blog-service/2022-11-21-apps

```diff
@@ -0,0 +1,17 @@
+---
+title: Sumo Logic Extension for Lambda Telemetry API
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - sumo logic
+  - aws
+  - lambda
+  - apps
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We are happy to announce that we have released an updated version of [Sumo Logic Lambda Extension](https://github.com/SumoLogic/sumologic-lambda-extensions) that supports collecting the telemetry data using the new [AWS Lambda Telemetry API](https://docs.aws.amazon.com/lambda/latest/dg/telemetry-api.html). The extension gives a per-invoke level view of the Lambda function and provides insights into the function and execution environment lifecycle. With this update, you can collect telemetry data including logs, metrics, and traces in JSON format directly from the Lambda execution level. For more information on how to start using it please refer to our [documentation](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/collect-aws-lambda-logs-extension/).
```

Comment:
For more information on how to start using it please refer to our documentation.
Suggestion: We should avoid using "please" in the user documentation.